src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -167,7 +167,12 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
           </Box>
         )}
         {required === false && (
-          <Typography variant="subtitle1">{optionalLabel}</Typography>
+          <Typography
+            variant="subtitle1"
+            sx={{ whiteSpace: 'nowrap' }}
```

Comment:
I think you may also want to apply [this](https://github.com/okta/okta-signin-widget/pull/3334/files#diff-abe05a12a330be7104db3deb82012989c081335e5b6a056d22d56e73e0a2083bR243) to this element below as well?